docroot/themes/humsci/humsci_basic/src/scss/components/_structured-card.scss

```diff
@@ -1,7 +1,6 @@
 .hb-card--structured {
   width: 100%;
   padding-bottom: hb-calculate-rems(28px);
-
```

Comment:
We wan't this space here. One of the linters requires a space above SCSS blocks.